lib/ansible/modules/cloud/amazon/ec2_asg.py

```diff
@@ -756,6 +779,20 @@ def get_launch_object(connection, ec2_connection):
             launch_object = {"LaunchTemplate": {"LaunchTemplateId": lt['LaunchTemplateId'], "Version": launch_template['version']}}
         else:
             launch_object = {"LaunchTemplate": {"LaunchTemplateId": lt['LaunchTemplateId'], "Version": str(lt['LatestVersionNumber'])}}
+
+        if mixed_instances_policy:
+            instance_types = mixed_instances_policy.get('instance_types', [])
+            policy = {
+                'LaunchTemplate': {
+                    'LaunchTemplateSpecification': launch_object["LaunchTemplate"]
```

Comment:
```suggestion
                    'LaunchTemplateSpecification': launch_object['LaunchTemplate']
```
Nit, but would you mind using single quotes here to be consistent with the style of the rest of the module?